domain/schema/controller_test.go

```diff
@@ -75,6 +75,9 @@ func (s *schemaSuite) TestDDLApply(c *gc.C) {
 		"external_controller",
 		"external_controller_address",
 		"external_model",
+
+		// Model list
```

Comment:
Not sure we need this...

Reply:
I've left this for now. I want to group schema tables in the tests as it's easier to reason about why they exist from a test perspective.